pkg/config/schema/kind/resources.gen.go

```diff
@@ -41,6 +41,7 @@ const (
 	Service
 	ServiceAccount
 	ServiceEntry
+	DNSName
```

Comment:
@howardjohn Looks this is not good to add a non exist schema here

Reply:
how about this

Reply:
I don't hate it.. we already have MeshNetworks, etc that are not real

Reply:
IC